netsim/devices/dellos10.py

```diff
@@ -3,31 +3,33 @@
 #
 from box import Box
 
-from . import _Quirks,need_ansible_collection
+from . import _Quirks,need_ansible_collection,report_quirk
 from ..utils import log
 from ..augment import devices
 
-def check_vlan_ospf(name: str, iflist: list, vname: str) -> None:
+def check_vlan_ospf(node: Box, vname: str) -> None:
+  name = node.name
+  iflist = node.get('interfaces',[])
   err_data = []
   for intf in iflist:
     if 'ospf' not in intf or intf.type != 'svi':
       continue
     err_data.append(f'Interface {intf.ifname} VRF {vname}')
 
   if err_data:
-    log.error(
-      f'Dell OS10 (node {name}) cannot run OSPF on virtual-network (SVI) interfaces',
-      category=log.IncorrectValue,
+    report_quirk(
+      f'node {name} cannot run OSPF on virtual-network (SVI) interfaces',
+      quirk='svi_ospf',
       more_data=err_data,
-      module='ospf')
+      node=node)
 
 class OS10(_Quirks):
 
   @classmethod
   def device_quirks(self, node: Box, topology: Box) -> None:
-    check_vlan_ospf(node.name,node.get('interfaces',[]),'default')
+    check_vlan_ospf(node,'default')
```

Comment:
Perhaps we should make the name of the default VRF a device property, and issue a warning (error?) if it's used in the topology?

As-is, a topology with a VRF called "default" would behave differently on OS10 versus (say) SR OS

Reply:
![image](https://github.com/user-attachments/assets/b3531388-47c6-41a5-b6fb-e95870ddfc0e)

Looks like I can create a vrf called "default", not sure if this is the base context